### PR TITLE
DiscoveryConfig - Make versions optional

### PR DESCRIPTION
**Pull request by zkayyali812**
Signed-off-by: Zachary Kayyali <zkayyali@redhat.com>
https://github.com/open-cluster-management/backlog/issues/12492

Minor update - fixes toggle of fields when adding/editing discoveryconfigs

### CODE REVIEW

File: frontend/public/locales/en/discovery.json [hunk 1]
```diff
@@ -32,7 +32,8 @@
     "discoveryConfig.lastActiveFilter.labelHelp": "Only discovered clusters active within this time period are found.",
     "discoveryConfig.lastActiveFilter.placeholder": "7",
     "discoveryConfig.discoveryVersions.label": "Red Hat OpenShift version",
-    "discoveryConfig.discoveryVersions.labelHelp": "Only Red Hat OpenShift clusters that match the selected versions are found.",
+    "discoveryConfig.discoveryVersions.labelHelp": "All Red Hat Openshift versions are included by default unless specified in this dropdown.",
+    "discoveryConfig.discoveryVersions.placeholder": "All available Red Hat Openshift versions are included by default",
```
Comment (swopebe) [inline frontend/public/locales/en/discovery.json:36]: All Red Hat OpenShift versions are included by default unless specified in this drop-down menu.

Changes to product representation, "drop-down" is correct and not an accessibility issue.

Comment (zkayyali812) [reply frontend/public/locales/en/discovery.json]: Updated!

**Review by jamestalton — APPROVED** [review]
lgtm

**Pull request merged**